modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -100,6 +100,21 @@ Currently known incompatible apps include:
 * richdocuments 2.7.0
 * onlyoffice 7.5.3
 
+==== Guests App Whitelist feature
+
+The _Guests App Whitelist feature_ is evaluating the whitelist stricter starting with release 0.12.0 . Therefore the administrator needs to make sure that essential core apps are added _before_ performing the upgrade:
+
+* admin_audit 
+* encryption
+* files_antivirus
+* firewall
+* ransomware_protection
+
+Any apps that provide access to external storage the guest users should have access too, need to be added as well like:
+
+* files_primary_s3
+* windows_network_drive
+
```

Comment:
```suggestion
==== Guests App Whitelist Feature

The _Guests App Whitelist feature_ evaluates the app whitelist stricter starting with release 0.12.0 . To overcome any issues related to this strict handling, an upgrade to Guests 0.12.1 is highly recommended as this release has a much wider range of default whitelisted apps included.

```

Reply:
The tarball does not get an upgrade, we need to add this to the known issues section.

@EParzefall @phil-davis mind for a language review?